src/replacer.rs

```diff
@@ -127,4 +143,31 @@ mod test {
 
         assert_eq!(replace_strings(&rules, &String::from("Me&You")), "MeYou");
     }
+
+    #[test]
+    fn test_remove_brackets_list_empty() {
+        let rules : Rules = Rules {
+            ..Default::default()
+        };
+
+        assert_eq!(replace_strings(&rules, &String::from("One: (content) should stay.")), "One: (content) should stay.");
+        assert_ne!(replace_strings(&rules, &String::from("Two: (content) should stay.")), "Two: should stay.");
+    }
+
+    #[test]
+    fn test_remove_brackets_list() {
+        let rules = Rules {
+            remove_brackets_list: vec![
+                Value::try_from([Value::try_from("(").unwrap(), Value::try_from(")").unwrap()]).unwrap(),
+                Value::try_from([Value::try_from("[").unwrap(), Value::try_from("]").unwrap()]).unwrap(),
+            ],
+            ..Default::default()
+        };
+
+        assert_eq!(replace_strings(&rules, &String::from("Three: (content) should be removed.")), "Three: should be removed.");
+        assert_eq!(replace_strings(&rules, &String::from("Four: [content (and nested one)] should be removed.")), "Four: should be removed.");
+        assert_eq!(replace_strings(&rules, &String::from("Five: (content (and nested one)) should be removed.")), "Five: should be removed.");
+        assert_eq!(replace_strings(&rules, &String::from("Six: (one) (two) and [three] 'and' should stay.")), "Six: and 'and' should stay.");
+        assert_ne!(replace_strings(&rules, &String::from("Seven: Test edge case (here).")), "Seven: Test edge case.");
```

Comment:
See above, not sure if we really need this.

Reply:
Removed both `_ne!` tests.